PencilCase/PencilCase/scripts/Navigator.ts

```diff
@@ -37,13 +38,17 @@ export class Navigator {
 
     public goHome = () => {
         Application.instance.activePage(new HomePage());
-        $(':mobile-pagecontainer').pagecontainer("change", "#" + Consts.Pages.HomePage.Id, { showLoadMsg: true });
+        $(':mobile-pagecontainer').pagecontainer("change", "#" + Consts.Pages.HomePage.Id, { showLoadMsg: true, transition: "flip" });
     }
 
-    public showConfirmDialog(header: string, content: string, confirm: () => void, cancel?: () => void) {
+    public showConfirmDialog(header: string, content: string, showConfirm: boolean, showCancel: boolean, confirm: () => void, cancel?: () => void, confirmButtonText?: string, cancelButtonText?: string) {
```

Comment:
Update confirm dialog, enable more capabilities, like show/hide confirm and cancel buttons, specify dialog title and content.